controllers/profiles/followUserV3.js

```diff
@@ -22,6 +23,11 @@ module.exports = async (req, res) => {
   )
     return ErrorResponse.e409(res, 'You have followed this user');
 
+  const targetUser = await UsersFunction.findUserById(User, user_id_followed);
+  if (!targetUser) {
+    return ErrorResponse.e404(res, 'User not found');
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on adding a check to see if the target user exists before proceeding with the follow operation. This will prevent potential errors and confusion in case an invalid `user_id_followed` is provided.



<!-- This is an auto-generated comment by OSS CodeRabbit -->